packages/material-ui/src/Menu/Menu.d.ts

```diff
@@ -7,12 +7,88 @@ import { TransitionHandlerProps, TransitionProps } from '../transitions/transiti
 
 export interface MenuProps
   extends StandardProps<PopoverProps & Partial<TransitionHandlerProps>, MenuClassKey> {
+  /**
+   * The DOM element used to set the position of the menu.
+   */
+  anchorEl?: PopoverProps['anchorEl'];
+  /**
+   * If `true` (Default) will focus the `[role="menu"]` if no focusable child is found. Disabled
+   * children are not focusable. If you set this prop to `false` focus will be placed
+   * on the parent modal container. This has severe accessibility implications
+   * and should only be considered if you manage focus otherwise.
+   */
   autoFocus?: boolean;
+  /**
+   * Menu contents, normally `MenuItem`s.
+   */
+  children?: React.ReactNode;
+  /**
+   * When opening the menu will not focus the active item but the `[role="menu"]`
+   * unless `autoFocus` is also set to `false`. Not using the default means not
+   * following WAI-ARIA authoring practices. Please be considerate about possible
+   * accessibility implications.
+   */
   disableAutoFocusItem?: boolean;
+  /**
+   * Props applied to the [`MenuList`](/api/menu-list/) element.
+   */
   MenuListProps?: Partial<MenuListProps>;
+  /**
+   * Callback fired when the component requests to be closed.
+   *
+   * @param {object} event The event source of the callback.
+   * @param {string} reason Can be: `"escapeKeyDown"`, `"backdropClick"`, `"tabKeyDown"`.
+   */
+  onClose?: PopoverProps['onClose'];
+  /**
+   * Callback fired before the Menu enters.
+   * @document
```

Comment:
I can't find any trace for the use case of this mention. What problem does it solve?

Reply:
It's a custom directive that is only used as a band-aid. It is required for props that are documented but not actually used. 

This will become obsolete eventually.